src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -195,9 +196,8 @@ public String getScript() {
         private transient long checkedTimestamp;
 
         private ShellController(FilePath ws) throws IOException, InterruptedException {
-            super(ws);
-        }
-
+                super(ws);
```

Comment:
Please do not change whitespace on unrelated lines.

Reply:
Sorry, will fix.